### PR TITLE
Switch from Travis to Github actions

### PR DESCRIPTION
**Pull request by bjgill**
Trello: https://trello.com/c/cnNEUjhz/2126-epic-migrate-travis-ci-usages-to-github-actions

Since Github actions can provide the same functionality as Travis CI it makes sense to migrate to it in order to:

* Reduce 3rd party dependency
* Benefit from faster build times due to lack of alphagov queue

Following the example of [supplier-frontend](https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/1289)

### CODE REVIEW

**Comment by bjgill** [comment]
Once this has been merged, I'll remove Travis from the list of required statuses and add Github actions.

File: requirements-dev.in [hunk 1]
```diff
@@ -1,16 +1,14 @@
 -c requirements.txt
 
-pip-tools>5.0.0
+pip-tools>=5.4.0
 
-coverage
 coveralls
```
Comment (domoscargin) [inline requirements-dev.in:5]: I see coveralls was removed on the supplier frontend example - is there a reason we're keeping it here?

**Review by domoscargin — APPROVED** [review]
LGTM - one comment, but not blocking.

File: requirements-dev.in (hunk 1 above)
Comment (bjgill) [reply requirements-dev.in]: Oops. Force pushed their removal.

**Comment by gidsg** [comment]
@bjgill can we remove the coveralls graphic from the readme?

**Comment by bjgill** [comment]
Good point. Let me merge this now to get Brett's PR unblocked, and then I'll sort that out.

**Pull request merged**